Assets/_BForBoss/_Core/Scripts/UserInterface/PauseViewBehaviour.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using Sirenix.OdinInspector;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace BForBoss
+{
+    public class PauseViewBehaviour : MonoBehaviour
+    {
+        [Title("Buttons")]
+        [SerializeField] private Button _resumeButton = null;
+        [SerializeField] private Button _resetButton = null;
+        [SerializeField] private Button _quitButton = null;
+        [SerializeField] private Button _settingsButton = null;
+
+        private Action OnSettingsPressed;
+
+        public void Initialize(Action onSettingsPressed)
+        {
+            OnSettingsPressed = onSettingsPressed;
+        }
+
+        private void ResetGame()
+        {
+            StateManager.Instance.SetState(State.PreGame);
+        }
+
+        private void ResumeGame()
+        {
+            StateManager.Instance.SetState(State.Play);
+        }
+
+        private void QuitGame()
+        {
+#if UNITY_EDITOR
+            EditorApplication.ExitPlaymode();
```

Comment:
Instead you can do `UnityEditor.EditorApplication.ExitPlaymode();`

Reply:
Fixed